quickstarts/microsoft.kubernetes/fleet/hubless-fleet-deploy.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "fleetName": {
+            "type": "string"
```

Comment:
as you default in bicep, you might do the same in ARM (`"defaultValue": "my-hubless-fleet"`), etc.